code/AssetLib/X/XFileParser.cpp

```diff
@@ -209,41 +183,26 @@ XFileParser::XFileParser(const std::vector<char> &pBuffer) :
             P1 += ofs;
             est_out += MSZIP_BLOCK; // one decompressed block is 32786 in size
```

Comment:
Odd that the spec seems to indicate it is 32KB + 12 bytes https://docs.microsoft.com/en-us/openspecs/exchange_server_protocols/ms-mci/3048c471-f739-4d30-bcb3-a962cfe84a69

Reply:
Right, but this is the original code which has worked for years.